auth/client/pom.xml

```diff
@@ -138,6 +141,11 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <environmentVariables>
+                        <CREDSTORE_PATH_ENV>PATH/TO/ELYTRON/CLIENT/DIR/target/credstore/mycredstore.cs</CREDSTORE_PATH_ENV>
```

Comment:
What is this needed for? Is this for tests?

Reply:
This is used for tests. 
Since the absolute path needs to be specified for the location of the credential store, I added an environment variable to specify the path. 
Maybe we can change it to something like what the OIDC tests do to resolve env variables.